HadesAPI/main.go

```diff
@@ -37,6 +38,14 @@ func main() {
 	gin.SetMode(gin.ReleaseMode)
 
 	r := gin.Default()
+	if cfg.AuthKey == "" {
```

Comment:
I just thought about it once more. Would it make sense to add a config to disable auth (especially during Development) - We can do that in a followup though

Reply:
I mean with this solution one can easily disable auth by just not setting any AuthKey in the ENV 🤔

Reply:
True that